tests/behat/features/asp/edit-cssn-roles-auth.feature

```diff
@@ -0,0 +1,19 @@
+@asp
+@api
+@javascript
+
+Feature: test ACCESS Support domain cssn roles webform editing
+
+  Scenario: Authenticated user tests editing cssn roles webform on ACCESS Support domain
+    Given I am logged in as a user with the "administrator" role
+    When I am on "/form/edit-your-cssn-roles"
+    Then I should see "Mentor"
+    Then I should see "Student"
+    Then I should see "Research Computing Facilitator"
+    Then I should see "Research Software Engineer"
+    Then I should see "CI Systems Engineer"
+    Then I should see "Researcher / Educator"
+    # NOT SURE IF I NEED TESTS BELOW
```

Comment:
It would be great to test that the form can be submitted successfully. Ideally we would also check if the new role is displaying on the community persona page.